### PR TITLE
Playbook to handle no entity matches IP.

### PR DESCRIPTION
**Pull request by matan-xmcyber**
New Incident Type for EDR incidents which only have IP info and not endpoint, such as CrowdStrike.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
1. When working with CrowdStrike the incidents were not smoothly handled because they are IP based. To handle IP requires setting the default playbook to be IP Enrichment, which is not a simple setting to do when there is no matching incident type.
2. once the playbook runs it can have errors when no entity is found in XM

## Screenshots
The new part of the Playbook:
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ x] Tests
- [ x] Documentation

### CODE REVIEW

**Comment by yaakovi** [comment]
@matan-xmcyber 
* Please add release notes (you can use the `demisto-sdk update-release-notes` command)
* update the playbook README with an updated image.
* you didn't connect the new incident type to the current classifier, was that intentional?

**Comment by matan-xmcyber** [comment]
1. Done
2. Done
3. None of the incident types have any reference to classifiers, am I wrong?

**Comment by yaakovi** [comment]
> 3. None of the incident types have any reference to classifiers, am I wrong?
from the classifier file:
image
according to your classifier settings, the `type` field returned from the integration will determine the incident type.
you should add the new incident type too.

**Comment by matan-xmcyber** [comment]
Oh, no those are incidents we fetch. This new incident type is not fetched
by us, it is meant to apply to incidents fetched by other integrations such
as CrowdStrike

File: Packs/XMCyber/ReleaseNotes/1_0_6.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Incident Types
+- **XM Cyber IP Incident**
+An incident type for incoming incidents that only have an IP address
+
+#### Playbooks
+- **IP Enrichment - XM Cyber**
+Bug fix
```
Comment (yaakovi) [inline Packs/XMCyber/ReleaseNotes/1_0_6.md:8]: - Fixed an issue where the playbook failed when the entity wasn't found in XM Cyber.

File: Packs/XMCyber/ReleaseNotes/1_0_6.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Incident Types
+- **XM Cyber IP Incident**
+An incident type for incoming incidents that only have an IP address
```
Comment (yaakovi) [inline Packs/XMCyber/ReleaseNotes/1_0_6.md:4]:
```suggestion
An incident type for incoming incidents that only have an IP address.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**